.github/workflows/ci.yml

```
@@ -23,12 +23,18 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Run unit tests
+      - name: Zet run permissions op script folder
+        uses: ./.github/actions/zet-run-bash-scripts-permission
+        with:
+          path-bash-file-or-folder: ./scripts/*
+
+      - name: Run jest unit tests
         run: |
-          ./scripts/unit-tests-verify.sh
-      - name: Validate Step Definitions
+          echo "### Jest Test Reports" >> $GITHUB_STEP_SUMMARY
+          ./scripts/unit-tests-verify.sh >> $GITHUB_STEP_SUMMARY
```

```suggestion
          ./scripts/unit-tests-verify.sh | sed '1d;$d' >> $GITHUB_STEP_SUMMARY
```

De eerste en laatste regel van deze test output zijn niet geschikt voor markdown. Kunnen we eruit filteren met sed.

---

aangepast